src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,77 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racinggame.dto.CarDto;
+import racinggame.dto.SnapShotDto;
+import racinggame.random.Radom;
+
+public class Cars {
+
+    private static final int DEFAULT_POSITION = 1;
+    private final SnapShotStore snapShotStore = new SnapShotStore();
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if(cars.isEmpty())throw new IllegalArgumentException("자동차가 비여 있을 수 없습니다.");
```

Comment:
```suggestion
        if(cars.isEmpty()) {
            throw new IllegalArgumentException("자동차가 비여 있을 수 없습니다.");
        }    
```

팀 컨벤션에 따라 다르겠지만,
개인적으로 한줄이라도 일관성을 위해 블록을 권장합니다.